app.py

```
@@ -1,20 +1,6 @@
-
-For API access, use `src/main.py`.
-For CLI access, use `cli.py`.
-"""
-
-# Import all necessary UI components from the refactored modules
-
-# Additional components are automatically imported via the @cl decorators
-# The event handlers defined in src/ui/app.py and src/ui/events.py will be
-# properly registered with Chainlit when this file is run.
+@cl.on_message
+async def handle_message(message: cl.Message):
```

Ruff (0.11.9)

Missing docstring in public function

(D103)